src/python/system/archive.py

```diff
@@ -295,7 +295,7 @@ def unpack(archive_path,
       logs.log_error(
           'Failed to extract everything from archive %s.' % archive_filename)
 
-  elif archive_type == ArchiveType.TAR or archive_type == ArchiveType.TAR_LZMA:
+  elif archive_type in (ArchiveType.TAR, ArchiveType.TAR_LZMA):
```

Comment:
Remove this, as i am changing this in another CL, otherwise your cl will conflict.

Reply:
sure, I will remove this part.